api/resolvers/item.js

```diff
@@ -1127,12 +1142,19 @@ export const createItem = async (parent, { forward, options, ...item }, { me, mo
     item.url = await proxyImages(item.url)
   }
 
-  const [result] = await serialize(
-    models,
+  const trx = [
     models.$queryRawUnsafe(`${SELECT} FROM create_item($1::JSONB, $2::JSONB, $3::JSONB, '${spamInterval}'::INTERVAL) AS "Item"`,
-      JSON.stringify(item), JSON.stringify(fwdUsers), JSON.stringify(options)),
-    ...trx)
-  item = Array.isArray(result) ? result[0] : result
+      JSON.stringify(item), JSON.stringify(fwdUsers), JSON.stringify(options))
+  ]
+  if (invoice) {
+    trx.unshift(models.$queryRaw`UPDATE users SET msats = msats + ${invoice.msatsReceived}`)
```

Comment:
ooof, lol, I just added the sats from the invoice to **_every_** user ... :see_no_evil:

Reply:
lol wouldn't it be *amazing* if postgres knew which user it was though lol